source/_patterns/01-elements/select/select.scss

```diff
@@ -39,12 +39,27 @@
 			}
 		}
 
-		&:required:invalid,
+		// TODO: Evaluate whether those could get moved to ../_form-elements.scss
```

Comment:
It's copy&paste from input, so why don't you move it into `../_form-elements.scss` and use it as a placeholder?